roles/openshift_node/defaults/main.yml

```diff
@@ -1,6 +1,6 @@
 ---
 os_firewall_allow:
-- service: OpenShift kubelet
+- service: Kubernetes kubelet
```

Comment:
@aveshagarwal In that case I would say that the user would be on their own...  We could have a flag for disabling modifying firewalld, and the user would be required to ensure the proper iptables rules are in place using their preferred mechanism (or use no host firewall).